taqueria-plugin-contract-types/src/generator/contract-parser.ts

```diff
@@ -96,7 +96,9 @@ const visitContractParameterEndpoint = (node: MMethod): TypedMethod[] => {
 	}
 
 	const nameRaw = node.annots?.[0];
-	const name = nameRaw?.startsWith('%') ? nameRaw.substr(1) : null;
+
+	// If the name is missing, then we assume its the main / default entrypoint
```

Comment:
Checking why this doesn't work: 

https://github.com/ecadlabs/taqueria/blob/2300f346cf72033d1f11ab12a8de658d1c04b3ba/taqueria-plugin-contract-types/src/generator/process.ts#L93